bin/newrelic-admin

```diff
@@ -71,11 +70,15 @@ USAGE
     # we need to make sure there is a command to run
     print_usage('run-perl') unless @ARGV;
 
-    use NewFangle qw(newrelic_init);
+    use NewFangle qw(newrelic_configure_log newrelic_init);
     require NewFangle::App;
     require NewFangle::Agent;
     require NewFangle::Agent::Config;
 
+    my $cfg = NewFangle::Agent::Config->local_settings;
+    newrelic_configure_log( $cfg->{'log_filename'} // 'stdout',
+        $cfg->{'log_level'} // 'info' );
```

Comment:
Same here: why is the default `info` if the default in the config object is `error`?